projects/plugins/crm/includes/ZeroBSCRM.DAL3.Obj.Companies.php

```diff
@@ -334,6 +334,33 @@ public function getFullCount(){
 
     }
 
+	/**
+	 * Retrieves the company ID by its name.
+	 *
+	 * @param  string $name  The name of the company whose ID is to be retrieved.
+	 * @return int|bool      Returns the ID of the company if found, false otherwise.
+	 *
+	 * @throws Exception     Catches and handles exceptions, logging SQL errors.
+	 *
+	 * @since  $$next-version$$
+	 */
+	public function get_company_id_by_name( $name ) {
+		global $ZBSCRM_t; // phpcs:ignore WordPress.NamingConventions.ValidVariableName.VariableNotSnakeCase
+		global $wpdb;
+
+		try {
+			$query  = $this->prepare( 'SELECT ID FROM ' . $ZBSCRM_t['companies'] . ' WHERE zbsco_name LIKE %s', $name ); // phpcs:ignore
+			$result = $wpdb->get_row( $query, OBJECT ); // phpcs:ignore
```

Comment:
I think it'd be better to just phpcs:ignore the relevant sniffs (moving up to a broader category for the second line, so that's `WordPress.DB.DirectDatabaseQuery` which covers two).
```suggestion
			$query  = $this->prepare( 'SELECT ID FROM ' . $ZBSCRM_t['companies'] . ' WHERE zbsco_name LIKE %s', $name ); // phpcs:ignore WordPress.NamingConventions.ValidVariableName.VariableNotSnakeCase
			$result = $wpdb->get_row( $query, OBJECT ); // phpcs:ignore WordPress.DB.DirectDatabaseQuery,WordPress.DB.PreparedSQL.NotPrepared
```